smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptJmesPathVisitor.java

```diff
@@ -196,10 +245,15 @@ public Void visitLiteral(LiteralExpression expression) {
                 executionContext = "\"" + expression.getValue().toString() + "\"";
                 break;
             case OBJECT:
+                @SuppressWarnings("unchecked")
+                Map<String, Object> objValue = (Map<String, Object>) expression.getValue();
```

Comment:
is there a method like expression.getObjectValue that avoids casting?

Reply:
done, small change in types from `ArrayList` to `List` to account for `expectArrayValue()` return type